book/02-git-basics/sections/recording-changes.asc

```diff
@@ -51,10 +52,10 @@ Untracked files:
 nothing added to commit but untracked files present (use "git add" to track)
 ----
 
-You can see that your new README file is untracked, because it's under the ``Untracked files'' heading in your status output.
-Untracked basically means that Git sees a file you didn't have in the previous snapshot (commit); Git won't start including it in your commit snapshots until you explicitly tell it to do so.
-It does this so you don't accidentally begin including generated binary files or other files that you did not mean to include.
-You do want to start including README, so let's start tracking the file.
+Você pode ver que o seu novo arquivo README é um arquivo não rastreado, porque está abaixo do subtítulo ``Untracked files'' na saída do seu status.
+"Não rastreado" basicamente significa que o Git vê um arquivo que você não tinha no snapshot (commit) anterior; o Git não vai passar a incluir o arquivo nos seus commits a não ser que você o mande fazer isso explicitamente.
+O comportamento do Git é dessa forma para que você não inclua acidentalmente arquivos binários gerados automaticamente ou outros arquivos que você não deseja incluir.
+Você _quer_ incluir o arquivo README, então vamos comeaçar a rastreá-lo.
```

Comment:
Não captei a ideia do sublinhado para quer. Não me parece a intenção do autor.

Reply:
A ideia aqui é dar ênfase. No original, o texto diz:

"You do want to include the README file..."

O "do" não é realmente necessário na frase, ele é usado para dar ênfase no fato de que nesse caso (o arquivo README) você quer adicionar ao stage, em oposição aos arquivos binários que ele menciona no parágrafo anterior.

Eu pensei em algumas opções para expressar essa ideia, e acabei optando pelo itálico. Outras opções seriam:

"Você de fato/realmente quer incluir o arquivo README..."
"Neste caso, você quer realmente incluir o arquivo README..."
"No caso do arquivo README, você quer de fato inclui-lo...."

O que você acha? Alguma dessas alternativas fica melhor?

Reply:
Legal. Não tinha visto dessa forma antes. Agora vejo. Obrigado por explicar.
Pessoalmente gosto da primeira alternativa. Mas não me oponho a nenhuma das opções, incluindo o itálico já no texto.